dpnp/backend/examples/example_bs_sycl.cpp

```diff
@@ -0,0 +1,97 @@
+#include <CL/sycl.hpp>
+#include <iostream>
+
+cl::sycl::event divide_sycl(double* price,
+                            double* strike,
+                            const size_t size,
+                            double* p_div_s,
+                            cl::sycl::queue q)
+{
+    double* P = sycl::malloc_shared<double>(size, q);
+    double* S = sycl::malloc_shared<double>(size, q);
+
+    P = price;
+    S = strike;
+
+    cl::sycl::event event = q.submit([&](sycl::handler &cgh) {
+	    cgh.parallel_for(size, [=](sycl::item<1> i) { p_div_s[i] = P[i] / S[i];});
+    });
+
+    cl::sycl::free(P, q);
+    cl::sycl::free(S, q);
+
+    return event;  
+}
+
+cl::sycl::event log_sycl(const size_t size,
+                         double* p_div_s,
+                         double* log_,
+                         cl::sycl::queue q)
+{
+    cl::sycl::event event = q.submit([&](sycl::handler &cgh) {
+	    cgh.parallel_for(size, [=](sycl::item<1> i) { log_[i] = cl::sycl::log(p_div_s[i]);});
+    });
+
+    return event;
+}
+
+
+void black_scholes(double* price,
+                   double* strike,
+                   const double rate,
+                   const double vol,
+                   const size_t size,
+                   cl::sycl::queue q)
+{
+    // ------------ dividing ----------
+    double(*p_div_s) = sycl::malloc_shared<double>(size, q);
+
+    cl::sycl::event event;
+    event = divide_sycl(price, strike, size, p_div_s, q);
+    event.wait();
+
+    for (int i = 0; i < size; ++i)
+        std::cout << price[i] << " / " << strike[i] << " = " << p_div_s[i] << "\n";
+
+    // ------------ log ----------
+    double(*log_) = sycl::malloc_shared<double>(size, q);
```

Comment:
Let's name the variable as in the python workload (`a`).
Why `double(*log_)`, why not `double* log_`?